### DISCUSSION TITLE
Could u make it to work with HDMI output?

### DISCUSSION THREAD

**Discussion by reklm-e** [FundamentalFrequency/.github#25] [General]
Is it possible to only ( or both) output to the Hdmi port without display.

**Comment by benpocalypse** [comment]
I have this working with my Raspberry Pi 3b+ with the Hyperpixel display and an HDMI monitor. Didn't have to do anything special, just used `arandr` to set it up once the PI booted up.

**Reply by reklm-e** [reply]
sorry for the question but whats arandr

**Reply by benpocalypse** [reply]
It's a program that lets you change the resolution and position of your connected displays.

See here: https://christian.amsuess.com/tools/arandr/

**Reply by reklm-e** [reply]
thanks
